.gitignore

```diff
@@ -9,6 +9,11 @@ logs/
 calibration_results/
 .coverage
 
+# Temp files
+**/*.tmp
+**/*.temp
+**/tmp.*
```

Comment:
I'm aware that similar files come from Qblox's Dummy cluster tests. They are also aware and they will be fixing it in the upcoming releases. Are there any other sources I may not know about ?

Reply:
No these are the ones from Qblox.